pii_check/pii_check_hook.py

```diff
@@ -165,7 +167,28 @@ def main():
 
     blocked_list = [blocked for blocked in args.blocked_list] if args.blocked_list else []
 
-    check_for_pii(args.url, API_KEY, enabled_entity_list, blocked_list)
+    try:
+        pii_results = [
+            result
+            for filename in args.filenames
+            for result in check_for_pii(os.path.abspath(filename), args.url, API_KEY, enabled_entity_list, blocked_list)
+        ]
+    except RuntimeError as e:
+        print(e)
+        print("If you get this message when running `pre-commit run -a` make sure to scan the files manually for PII instead of using this hook.")
+        return 2
+
+    if pii_results:
+        for result in pii_results:
+            print(
+                f"""Found PII [{result.entity_type}]: File "{result.filename}", line {result.line}, at index {result.start_index}:{result.start_index+ result.entity_length}"""
```

Comment:
This output message is "Visual Studio Code"-friendly and it brings users to the exact line where PII was detected when clicked.